### PR TITLE
Avatar mc

### PR DESCRIPTION
**Pull request by JoeyDarbyshire**
Completes #10 
#32 Still not fixed, placed on the back-burner for now

### CODE REVIEW

File: browser/utils.js [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
+  scene.appendChild(body);
+  head.setAttribute('id', user.id);
+  body.setAttribute('id', `${user.id}-body`);
+  head.setAttribute('minecraft-nickname', user.color);
+  head.setAttribute('minecraft', 'skinUrl: ../../images/3djesus.png;');
+  body.setAttribute('minecraft', 'skinUrl: ../../images/3djesus.png;  component: body; heightMeter: 0.4');
+  head.setAttribute('position', `${user.x} ${user.y} ${user.z}`);
+  body.setAttribute('position', `${user.x} ${user.y} ${user.z}`);
+  head.setAttribute('rotation', `${user.xrot} ${user.yrot} ${user.zrot}`);
+  head.setAttribute('rotation', `0 ${user.yrot} 0`);
```
Comment (parky22) [inline browser/utils.js]: should this be body.setAttribute?

File: browser/utils.js [hunk 2]
```diff
@@ -1,13 +1,19 @@
...
-  avatar.setAttribute('rotation', `${user.xrot} ${user.yrot} ${user.zrot}`);
-  return avatar;
+  const head = document.createElement('a-minecraft');
+  const body = document.createElement('a-minecraft');
+  scene.appendChild(head);
+  scene.appendChild(body);
+  head.setAttribute('id', user.id);
+  body.setAttribute('id', `${user.id}-body`);
+  head.setAttribute('minecraft-nickname', user.color);
+  head.setAttribute('minecraft', 'skinUrl: ../../images/3djesus.png;');
```
Comment (parky22) [inline browser/utils.js:10]: is this just getting the head? just curious how that's working since for body you had to say component:body

Comment (JoeyDarbyshire) [reply browser/utils.js:10]: It defaults to head, so I only wanted to define the attribute if I wanted it to be anything other than the head

**Review by bethqiang — COMMENTED** [review]
Looks good to me! The only thing I might say as a future thing would be to actually connect the head and the body (by making them mixins, perhaps, or just making them into one entity) so that we don't actually have to manipulate them separately. But I'm fine with how things look as of now!

**Comment by bethqiang** [comment]
Also I definitely meant to press "approve" on that comment ^ but...uhh...that didn't happen. I approve!

**Review by bushidocodes — APPROVED** [review]

**Pull request merged**